docs/tutorial.rst

```diff
@@ -0,0 +1,127 @@
+
+Once the keyspace is created the user needs to create a table and load it. To do that the
+user needs to run the **.cql** file under the **dask-cassandra-loader/docs** directory as follow:
+
+.. code-block:: bash
+
+    cqlsh --keyspace=tutorial -f tutorial.cql
+
+Once the table is loaded, the user will have a table called **tab1** with the following schema: 
+
+.. code-block:: sql
+
+  create table tab1(id int, year int, month int, day int, timest timestamp, lat float, lon float, PRIMARY KEY((id, year, month)));
+  
+The loaded data has two partitions due two distinct months.
+
+
+Dask cassandra loader
+=====================
+
+The first step to load a table from Cassandra into a Dask data-frame is to create :class:`dask_cassandra_loader.loader.Loader`.
+To do that the user should do the following:
+
+.. code-block:: python
+
+  from dask_cassandra_loader import Loader
+  
+  dask_cassandra_loader = Loader()
+
+
+Connect to Cassandra
+--------------------
+
+With the loader the user is then able to set a connection to an existent Cassandra cluster.
+In this example we assume the user is connecting to local cluster using the default credentials.
+
+.. code-block:: python
+
+  keyspace = 'tutorial'
+  clusters = ['127.0.0.1']
+
+  dask_cassandra_loader.connect_to_cassandra(clusters, keyspace, username='cassandra', password='cassandra')
+
+
+Connect to Dask
+---------------
+
+Before a table is loaded it is necessary to connect to a Dask Cluster. For testing proposes
+it might be handy to have the option to create a **LocalCluster**. Both options are supported as
+the following examples will show.
+
+To create and connect to a local Dask cluster you use the following code:
+
+.. code-block:: python
+
+  dask_cassandra_loader.connect_to_local_dask()
+
+To connect to a remote cluster you use the following code:
+
+.. code-block:: python
+
+  cluster = "host1.domain.nl:9091"
+  dask_cassandra_loader.connect_to_dask(cluster):
+
+
+Read Table
+----------
+
+In this example the user will load table *tab1*, project columns *id*, *year*, *month*, *day*,
+have a predicate on column *day* (*day = 18*) and only select the partitions for which *id in [18]*,
+*year in [2018]* and *month in [11]*. In this example, it is requested to not load all partitions in
+case the query qualifies all of them for loading. For more details about the function, the user should
+read :doc:`API documentation <dask_cassandra_loader.loader.Loader.load_cassandra_table>`.
+
+.. code-block:: python
+
+  dask_cassandra_loader.load_cassandra_table('tab1',
+                                             ['id', 'year', 'month', 'day'],
+                                             [('day', 'equal', [8])],
+                                             [('id', [18]), ('year', [2018]),
+                                              ('month', [11])],
+                                             force=False)
+
+  table = dask_cassandra_loader.keyspace_tables['tab1']
```

Comment:
Why do I have to use keyspace_tables? Why doesn't the `load_cassandra_table` just return a dask dataframe

Reply:
The idea is that you can easily find the table without having to load it again. At the same time, we save the metadata which is handy to know what query did we used to load the data. Do not forget the loaded table is a view (more precisely a materialized view) over the original data. We never load the entire table since it is too much data. 

In case the user wants to load the table with a new view, (s)he has to unload it and re-load it again. We only accept one view at the time per table.

Reply:
The table object can be used for the metadata, I am OK with that. So the dataframe is wrapped in a table object.

Is there any restriction on the number of views of the same table, other than the libraries use of a dictionary?
If not then the  `load_cassandra_table()` should return a table object. The library would then have a simpler API.

Reply:
No, I can easily change the function to return table:
```
        table.load_data(self.cassandra_con, loading_query)
        self.keyspace_tables[table_name] = table
        return
```

becomes:

```
        table.load_data(self.cassandra_con, loading_query)
        return table
```

Then the unload table will disappear.

Reply:
Covered by 694e223 and 8cb42fb